specification/postgresql/resource-manager/Microsoft.DBforPostgreSQL/preview/2021-06-01-preview/examples/QueryTextsGet.json

```diff
@@ -0,0 +1,22 @@
+{
+  "parameters": {
+    "api-version": "2021-06-01-preview",
+    "subscriptionId": "ffffffff-ffff-ffff-ffff-ffffffffffff",
+    "resourceGroupName": "testResourceGroupName",
+    "serverName": "testServerName",
+    "queryId": 1
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "queryTexts-1",
```

Comment:
>"queryTexts-1 [](http://example.com/codeflow?start=14&length=13)

It has to be a full resource id. e.g.
/subscriptions/ffffffff-ffff-ffff-ffff-ffffffffffff/resourceGroups/testResourceGroupName/providers/Microsoft.DBforPostgreSQL/flexibleServers/testServerName/queryTexts/1